colin/core/checks/envs.py

```diff
@@ -16,6 +16,7 @@
 
 import re
 
+from colin.core.target import inspect_object
```

Comment:
Only small nit: can we use relative imports for all? => `s/colin.core.target/..target`

Reply:
My pycharm uses absolute paths, not sure why. I left it there on purpose and wondered whether you'll notice it :P